x-pack/filebeat/input/httpjson/input.go

```diff
@@ -312,12 +312,14 @@ func (in *HttpjsonInput) processHTTPRequest(ctx context.Context, client *http.Cl
 		switch obj := m.(type) {
 		// Top level Array
 		case []interface{}:
+			in.log.Info("Processed:", len(obj))
```

Comment:
How about `Debug` level for these? Filebeat ships with Info level enabled by default. It looks like this will get logged a lot.